kiwi/package_manager/apt.py

```diff
@@ -513,3 +501,54 @@ def _package_requests(self) -> List:
         items = self.package_requests[:]
         self.cleanup_requests()
         return items
+
+    def _run_bootstrap_scripts(
+        self, solved_debootstrap_packages: List[str], only_for: List[str] = []
+    ):
+        # FIXME: weird stuff
+        # These env variables are required for the package scripts
+        # to work. I don't understand their meaning and I'm not sure
+        # if hardcoding this for libc6 is something we can afford ?
+        # Help is appreciated
+        self.command_env['DPKG_MAINTSCRIPT_NAME'] = 'true'
+        self.command_env['DPKG_MAINTSCRIPT_PACKAGE'] = 'libc6'
```

Comment:
@IsaacJT As we discussed I created a refactor and implementation for the apt based bootstrap such that all package scripts from the resolved apt list are taken into account.

At this point however I needed two weird environment variables and I had to set libc6 as DPKG_MAINTSCRIPT_PACKAGE otherwise I got usr-merge complains. I really have no idea what these variables really do and my knowledge on the inner works of Debian/Ubuntu packaging is really bad.

I wanted to ask if you can give this PR a try if it would also fix the issue you found and maybe check if the above is ok as a permanent in code setup or if this is all wrong. I believe the way how I set the variables is not a good way to handle the issue that comes up if you disable them. Feel free to run the process without these variables so you can see the errors from apt.

Thanks much

Reply:
AFAIK `DPKG_MAINTSCRIPT_NAME` should be the name of the script (e.g. `postinst`, etc.) and `DPKG_MAINTSCRIPT_PACKAGE` should be the name of the package that the scripts belong to. I would have to double check this though...

Haven't had time to a have a proper look through yet.

Reply:
Ah ok

Reply:
As we run the scripts manually the setting of just `true` would then be ok ?

Reply:
Please also note that `libc6` was the only package in the collection of all packages that complains about the missing variables DPKG_MAINTSCRIPT_*

Reply:
I guess I simply do not understand the concept behind them

Reply:
I updated the comment for this variable settings. I could not find any bad or negative side effect of setting them during the bootstrap of a Debian based system. I tested Ubuntu and Debian and the integration tests pass and boots. Thus I leave it up to the experts if this needs to be changed or can harm the bootstrap procedure.

Reply:
Maybe `DPKG_MAINTSCRIPT_PACKAGE` should be set in the for block below so that it represents the name of the package that the script is currently being run for? And `DPKG_MAINTSCRIPT_NAME` could be set 
to the script name immediately before calling the script, e.g. 

```
if os.path.exists(script_pre):
    self.command_env['DPKG_MAINTSCRIPT_PACKAGE'] = 'preinst'
    ...
```

I think this would more closely emulate the environment that the scripts would normally be run in